viewing_party/party.py

```diff
@@ -1,23 +1,360 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    create_movie(title, genre, rating):
+    
+    takes three parameters: `title`, `genre`, `rating`
+    If those three attributes are truthy, then return a dictionary. This dictionary should...
+    Have three key-value pairs, with specific keys
+    The three keys should be `"title"`, `"genre"`, and `"rating"`
+    The values of these key-value pairs should be appropriate values
+    If `title` is falsy, `genre` is falsy, or `rating` is falsy, this function should return `None`
+
+    returns movie_dictionary
+    '''
+    if (not title) or (not genre) or (not rating):
+        # here we check if we have any of the attributes are falsy, we return None 
+        return None
+
+    movie_dictionary = {} #create empty dictionary
+
+    movie_dictionary["title"] = title #add key "title" with corresponding value
+    movie_dictionary["genre"] = genre #add key "genre" with corresponding value
+    movie_dictionary["rating"] = rating #add key "rating" with corresponding value
+   
+    return movie_dictionary

```

Comment:
This works well! One cool thing we could leverages is that if a function doesn't explicitly return a value, it returns None by default. As a result, we could have cut down the code just a bit by running the function only if all the values are truthy. An example would be: 

     if title and genre and rating:
          movie_dictionary = {"title" : title, "genre" : genre, "rating" : rating}
          return movie_dictionary